### PR TITLE
Added new WSDLS with response verification in compatibility with the …

### PR DESCRIPTION
**Pull request by AxelSkrauba**
…above.

### CODE REVIEW

File: django_afip/models.py [hunk 1]
```diff
@@ -88,8 +88,12 @@ def check_response(response) -> None:
...
     message.
     """
-    if response.Errors:
-        raise exceptions.AfipException(response)
+    if "Errors" in dir(response):
+        if response.Errors:
+            raise exceptions.AfipException(response)
+    elif "errorConstancia" in dir(response):
+        if response.errorConstancia:
+            raise exceptions.AfipExceptionWSSR(response)
```
Comment (WhyNotHugo) [inline django_afip/models.py:96]: Just raise `AfipException` here. I don't think there's a need for a separate error class.

File: django_afip/exceptions.py [hunk 2]
```diff
@@ -17,6 +17,20 @@ def __init__(self, response):
...
+
+    def __init__(self, response):
+        Exception.__init__(
+            self,
+            "Error Constancia: {}".format(
+                response.errorConstancia.error[0],
+            ),
+        )
+
+
```
Comment (WhyNotHugo) [inline django_afip/exceptions.py:33]: No need for a separate error class here.

**Review by WhyNotHugo — COMMENTED** [review]
I don't think we need a separate error class here. Everything else looks good.

File: django_afip/models.py (hunk 1 above)
Comment (AxelSkrauba) [reply django_afip/models.py]: Perfect, changes implemented.

File: django_afip/models.py [hunk 3]
```diff
@@ -88,8 +88,12 @@ def check_response(response) -> None:
     This method checks if responses have an error, and raise a readable
     message.
     """
-    if response.Errors:
-        raise exceptions.AfipException(response)
+    if "Errors" in response:
+        if response.Errors:
+            raise exceptions.AfipException(response)
+    elif "errorConstancia" in response:
+        if response.errorConstancia:
```
Comment (AxelSkrauba) [inline django_afip/models.py:95]: At this stage it seems that there is redundancy in the validations, but first the type of key for the error is determined according to the query, and then if it contains any value.

**Comment by WhyNotHugo** [comment]
Thanks!

**Pull request merged**